### PR TITLE
create file with user and group

### PR DESCRIPTION
**Pull request by n1mh**
Default user/group for the new file was user/user but, when there's a LDAP or Active Directory managing users and groups, the assignment failed. This code get the default group ID for the user and assign it to the file.

### CODE REVIEW

File: providers/printers.rb [hunk 1]
```diff
@@ -79,8 +79,14 @@ def delete_printer(prt_name)
       printers_list.each do |printer|
         Chef::Log.info("Processing printer: #{printer.name}")
 
+        # HP printers are bad labeled in database
```
Comment (alfem) [inline providers/printers.rb]: Could you take these unrelated lines out of this PR?

Comment (n1mh) [inline providers/printers.rb]: changed.
